src/uniprotkb/components/entry/similar-proteins/SimilarProteins.tsx

```diff
@@ -112,19 +110,24 @@ const SimilarProteins: FC<{
                     )
                   )}
                   {/* TODO: This query doesn't seem to work currently */}
-                  <Button
-                    onClick={() => {
-                      history.push({
-                        pathname: LocationToPath[Location.UniProtKBResults],
-                        search: `query=(${data?.results
-                          .filter(({ entryType }) => entryType === clusterType)
-                          .map(({ id }) => `uniref_cluster_100:${id}`)
-                          .join(' OR ')})`,
-                      });
+                  <Link
```

Comment:
I meant, keep the `Button` component from Franklin, but make it be a react-router-dom `Link` by setting `element={Link}`

Reply:
Hadn't realised this was an option.

Reply:
Does that not work using `Link`?

Reply:
https://github.com/ebi-uniprot/uniprot-website/pull/264#discussion_r567015967

Reply:
Done now - don't think I'd fully grasped the way to use Button with the new implementation